packages/javascript/bh-shared-ui/src/constants.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { ActiveDirectoryRelationshipKind } from '.';
+
+export const EdgesWithComposition = [
+    ActiveDirectoryRelationshipKind.GoldenCert,
```

Comment:
Can we generate this via the cue files?

Reply:
I guess if its only used in one place, potentially not necessary

Reply:
I thought about that and had that working but then I realized we would only use it in the UI since the backend needs mapping to the handler and I don't want to define the handlers from the go code in the cue schema

Reply:
Actually I might have to pull these from a list defined in ad.cue to avoid the circular dependency that's causing tests to fail ;s